src/main/resources/assets/js/components/AppListComponent.jsx

```diff
@@ -74,6 +74,7 @@ var AppListComponent = React.createClass({
           <col style={{width: "18%"}} />
           <col style={{width: "18%"}} />
           <col style={{width: "18%"}} />
+          <col style={{width: "156px"}} />
```

Comment:
Let's keep these so they always add up to `100%`. But replacing the status column, this shouldn't be a problem anymore.